### PR TITLE
Fixed search scroll issue

### PR DESCRIPTION
**Pull request by sanjam-deriv**

### CODE REVIEW

**Review by adam-deriv — APPROVED** [review]

File: SelectRequestInput.jsx [hunk 1]
```diff
@@ -48,34 +48,36 @@ const SelectRequestInput = ({ selected, setSelected, handleChange, selected_valu
...
+                            <div className={styles.dropdownSelect}> Select API Call - Version 3</div>
+                            <div className={styles.dropdownStart}>ALL CALLS</div>
+                            {playground_requests
+                                .filter(option => {
+                                    if (option.title.toLowerCase().includes(searchResults.toLowerCase())) {
+                                        return option;
+                                        // eslint-disable-next-line no-else-return
+                                    } else {
+                                        return;
+                                    }
```
Comment (matin-binary) [inline SelectRequestInput.jsx:61]: You should simplify this if statement

File: SelectRequestInput.jsx [hunk 2]
```diff
@@ -53,12 +53,8 @@ const SelectRequestInput = ({ selected, setSelected, handleChange, selected_valu
...
                                 .filter(option => {
-                                    if (option.title.toLowerCase().includes(searchResults.toLowerCase())) {
-                                        return option;
-                                        // eslint-disable-next-line no-else-return
-                                    } else {
-                                        return;
-                                    }
+                                    const result = (option.title.toLowerCase().includes(searchResults.toLowerCase())) ?  option  : null;
+                                    return result; 
                                 })
```
Comment (matin-binary) [inline SelectRequestInput.jsx:58]: Using arrow function, you don't need to define a variable to return.
Suggestion:
`
playground_requests.filter(option => (option.title.toLowerCase().includes(searchResults.toLowerCase())) ?  option  : null
)
`

**Review by matin-binary — APPROVED** [review]
LGTM

**Comment by aswathy-deriv** [comment]
Please resolve the conflicts and move from draft @sanjam-deriv

**Review by aswathy-deriv — APPROVED** [review]

**Pull request merged**